Dockerfile

```diff
@@ -1,68 +1,79 @@
 FROM registry.access.redhat.com/ubi8/ubi
 MAINTAINER Hazelcast, Inc. Integration Team <info@hazelcast.com>
 
-ENV HZ_HOME="/opt/hazelcast" \
-    HZ_CP_MOUNT="${HZ_HOME}/external" \
-    LANG="en_US.utf8" \
-    USER_NAME="hazelcast" \
-    USER_UID=10001
-
-ENV HZ_VERSION 4.0
-
+# Versions of Hazelcast and Hazelcast plugins
+ARG HZ_VERSION=4.0.1
```

Comment:
@lazerion and @Holmistr,
I do not have detailed info about new release script but version update mechanism might need to change, FYI. For instance, [old script ](https://github.com/hazelcast/DevOps/blob/master/build-tool/build.xml#L144) searches and modifies **ENV** not **ARG**.

Reply:
Thanks @hasancelik I think this confirms your expectation regarding **ENV** update
https://github.com/hazelcast/hazelcast-pipeline-library/blob/master/test/com/hazelcast/qe/pipeline/distribution/dockerfiles/DockerfileUpdateStepTest.groovy#L70
cc @Holmistr